build.gradle

```diff
@@ -6,21 +6,20 @@ apply from: 'publish.gradle'
 
 group = 'com.github.backtrace.io'
 
-version = '0.9.0'
+version = '0.9.1'
 
 repositories {
     jcenter()
 }
 
 dependencies {
-    implementation 'com.github.backtrace-labs.backtrace-java:backtrace-java:0.9.1'
-    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.9'
```

Comment:
We don't need this dependency anymore?

Reply:
@BartoszLitwiniuk tested it builds and is able to be imported without this dependency. Let's remove it!

Reply:
I was testing everything on independent project without this dependency and everything was working. I will also test it from staging to make sure that we don't break anything.